### DISCUSSION TITLE
[求助]遇到这种情况怎么办

### DISCUSSION THREAD

**Discussion by ShuiJingOfficial** [share121/inter-knot#1433] [其他]
![e23915c4c3ddcea019a6c54dcf5487db](https://github.com/user-attachments/assets/6637817e-46be-4771-98ad-b57ef8f87962)
有没有好心绳匠V我50回回血

**Comment by 13579yangh** [comment]
下班了，不听

**Comment by suiry-D** [comment]
这个委托接不了

**Comment by Kujojolen** [comment]
这个委托我接了！上米崩坏三的客服要求换就行，只要不抽应该就可以，如果不行，那只能去对应的池子捞点老婆了

**Comment by Mayf1ower** [comment]
这委托我，我不接了

**Comment by ghost** [comment]
那我这个算什么
![Screenshot_2024-02-17-23-22-46-883_com miHoYo enterprise NGHSoD](https://github.com/user-attachments/assets/754e4a24-38ef-4540-ad1c-7d344911597e)
